python/ct/crypto/verify_test.py

```diff
@@ -349,6 +253,138 @@ def test_verify_embedded_scts_with_preca_valid_signature(self):
                     ['test-embedded-with-preca-cert.pem', 'ca-cert.pem'])
         self.assertEqual(result, [(sct, True)])
 
+
+class LogVerifierRsaTest(LogVerifierTest, unittest.TestCase):
+    default_sth = client_pb2.SthResponse()
+    default_sth.tree_size = 1130
+    default_sth.timestamp = 1442500998291
+    default_sth.sha256_root_hash = (
+        "58f4e84d26f179829da3359a23f2ec519f83e99d9230aad6bfb37e2faa82c663"
+        ).decode("hex")
+
+    default_sth.tree_head_signature = (
+        "040101002595c278829d558feb560c5024048ce1ca9e5329cc79b074307f0b6168dda1"
+        "5b27f84c94cce39f8371aa8205d73a7101b434b6aeaf3c852b8471daa05d654463b334"
+        "5103c7406dbd4642c8cc89eababa84e9ad663ffb3cc87940c3689d0c2ac6246915f221"
+        "5da254981206fed8505eed268bcc94e05cd83c8e8e5a14407a6d15c8071fabaed9728a"
+        "02830c6aef95969b0576c7ae09d50bdfc8b0b58fa759458c6d62383d6fe1072c0da103"
+        "1baddfa363b58ca78f93f329b1f1a15b9575988974dcba2421b9a1bb2a617d8b3f4046"
+        "ead6095f8496075edc686ae4fa672d4974de0fb9326dc3c628f7e44c7675d2c56d1c66"
+        "32bbb9e4a69e0a7e34bd1d6dc7b4b2").decode("hex")
+
+    default_key_info = client_pb2.KeyInfo()
+    default_key_info.type = client_pb2.KeyInfo.RSA
+    default_key_info.pem_key = (
+        "-----BEGIN PUBLIC KEY-----\n"
+        "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAolpIHxdSlTXLo1s6H1OC\n"
+        "dpSj/4DyHDc8wLG9wVmLqy1lk9fz4ATVmm+/1iN2Nk8jmctUKK2MFUtlWXZBSpym\n"
+        "97M7frGlSaQXUWyA3CqQUEuIJOmlEjKTBEiQAvpfDjCHjlV2Be4qTM6jamkJbiWt\n"
+        "gnYPhJL6ONaGTiSPm7Byy57iaz/hbckldSOIoRhYBiMzeNoA0DiRZ9KmfSeXZ1rB\n"
+        "8y8X5urSW+iBzf2SaOfzBvDpcoTuAaWx2DPazoOl28fP1hZ+kHUYvxbcMjttjauC\n"
+        "Fx+JII0dmuZNIwjfeG/GBb9frpSX219k1O4Wi6OEbHEr8at/XQ0y7gTikOxBn/s5\n"
+        "wQIDAQAB\n"
+        "-----END PUBLIC KEY-----\n")
+
+
+class LogVerifierEcdsaTest(LogVerifierTest, unittest.TestCase):
```

Comment:
Surprised and impressed this is working :)
Let's verify with somebody more knowledgable in Python that this is the right approach to base two separate tests on the same code.